core/templates/domain/exploration/exploration-id-validation.service.ts

```diff
@@ -50,6 +51,24 @@ export class ExplorationIdValidationService {
           return response.correctness_feedback_enabled;
         });
   }
+
+  async categoryNotDefault(explorationId: string): Promise<boolean> {
```

Comment:
Bit confused about what this function is expected to do, function names are expected to start with a verb because it performs an "action". Can we rename this accordingly?